packages/catalog-process/src/services/catalogService.ts

```diff
@@ -455,7 +457,7 @@ export function catalogServiceBuilder(
       );
     },
 
-    async updateDescriptor(
+    async updateDraftDescriptor(
```

Comment:
Renaming. The former `updateDescriptor` can only update a draft descriptor, so it could be renamed to `updateDraftDescriptor`. With this change, we can add an `updateDescriptor` for this endpoint.